### DISCUSSION TITLE
How to use the package in plain Javascript?

### DISCUSSION THREAD

**Discussion by codervince** [gillescoolen/deepl-client#21] [General]
Doesn't seem as simple as 
const translate = require("deepl-client");
Could you add some info on how to use this module in Plain JS.

**Comment by gillescoolen** [comment]
Sorry for the delay, I'll try to write a guide during the weekend!

**Reply by gillescoolen** [reply]
Hi @codervince, I (finally) got around to writing a guide in plain javascript. You can use the package like this:

First, run `npm install deepl-client` in your terminal.
Then, import the package and call the desired function, like this:

```js
// Import the package
const client = require('deepl-client');

// Create a parameter object.
const parameters = {
  auth_key: 'your-auth-key,
  text: 'This is a sentence.',
  target_lang: client.Language.French,
}

// Use the translate function and log the response.
client.translate(parameters).then(data => console.log(data));
```

The snippet above will result in the following log:

```bash
> node index.js
{
  translations: [ { detected_source_language: 'EN', text: "C'est une phrase." } ]
}
```

I hope this clarifies how to use the package. Sorry for taking such a long time!
